itest/rust/Cargo.toml

```diff
@@ -10,8 +10,9 @@ publish = false
 crate-type = ["cdylib"]
 
 [features]
-default = []
-codegen-full-experimental = ["godot/__codegen-full", "godot/experimental-godot-api"]
+default = ["codegen-full"]
+codegen-full = ["godot/__codegen-full"]
+codegen-full-experimental = ["codegen-full", "godot/experimental-godot-api"]
```

Comment:
Please don't change the default, the whole point of _not_ having full codegen is to allow for fast local development and CI.

Also keep changes to CI only on a minimal level (`godot` -> `itest` feature), would be good to not burden existing CI jobs with more workload.

Reply:
The problem I'm trying to solve here is that rust-analyzer (and other IDE tools possibly as well) enable all default features for all workspace crates, which causes `godot/__codegen-full` to be enabled but `itest/godegen-full` remains disabled. This causes errors during development. In `check.sh`, default features are disabled.

Reply:
The original design was that itest would _never_ use full codegen. (The fact that it did through `dev-dependencies` is a bug).

It's this PR which introduces the differentiation between minimal+full codegen, and with it some problems.

It probably makes sense to have it so that audio tests don't always run, but it seems this has quite a complexity cost:
- more features to be propagated
- another dimension in which tests can run differently (next to platform, API level, threads...) -> combinatoric explosion

I _guess_ it's fine, as I don't have a better solution right now, but I'll probably need to get back to this in the future.

Reply:
Audio tests only run in the experimental Godot itest in CI. This codegen issue comes from the changing `IPrimitiveMesh` trait, depending on the scope of the codegen. An alternative would be to add the `Material` class to the minimal codegen.